### PR TITLE
[ja] Translate pricing

### PR DESCRIPTION
**Pull request by riywo**

### CODE REVIEW

**Comment by riywo** [comment]
@poppoerika Please review.

~~Note: "Will it blend?" section is not translated yet because it's not clear for me what "blend/blender" means. Please let me know more details so that I can update this PR.~~

"Will it blend?" part is a high context joke. We don't have to translate it to the other languages.

File: i18n/ja/docusaurus-plugin-content-docs/current/pricing.md [hunk 1]
```diff
@@ -2,76 +2,70 @@
...
+_詳細は[Wolfram Alpha](https://www.wolframalpha.com/input?i=5000%2Fsecond+*+1KB++*+%24.15%2FGB+to+%24%2Fmonth) の計算をご覧ください。_
 
-##### You have a 20 RPS Lambda that caches 1KB objects. Momento would cost you:
+##### 20 リクエスト毎秒のLambda があって、1KB のオブジェクトをキャッシュするとします。Momento の課金は以下の様になります:
 
-`20 requests/second * 86400 seconds/day * 30.5 days/month * 1KB` = **$.38/month**
+`20リクエスト毎秒 * 86400秒/日 * 30.5日/月 * 1KB` = **$.38/月**
 
-_Note: Without the free tier, this workflow would cost $7.88/month_
+_注: 無料枠なしだと、このワークフローは$7.88/月になっていました。_
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/pricing.md]: 無料枠なしだと、このワークフローは$7.88/月になります。

File: i18n/ja/docusaurus-plugin-content-docs/current/pricing.md [hunk 2]
```diff
@@ -2,76 +2,70 @@
...
 
-_See the calculation in [Wolfram Alpha](https://www.wolframalpha.com/input?i=%28%2820+requests%2Fsecond+*+1KB%2Frequest%29+-+50GB+%2Fmonth%29+*+%24.15%2FGB+to+%24%2Fmonth) for more detail._
+_詳細は[Wolfram Alpha](https://www.wolframalpha.com/input?i=%28%2820+requests%2Fsecond+*+1KB%2Frequest%29+-+50GB+%2Fmonth%29+*+%24.15%2FGB+to+%24%2Fmonth) の計算をご覧ください。_
 
-### FAQ's 
+### よくある質問
 <details open>
-  <summary>Is it really just $.15/GB transferred in and out of Momento? What else do you charge for …</summary>
+  <summary>Momento は本当に$.15/GB のデータ転送の出入りだけなのですか？それ以外にお金がかかるのは？</summary>
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/pricing.md]: それ以外にお金がかかるものはありますか？

File: i18n/ja/docusaurus-plugin-content-docs/current/pricing.md [hunk 3]
```diff
@@ -2,76 +2,70 @@
...
  </details>
 
 <details>
-<summary>Can I run a production app for free on Momento?</summary>
-Absolutely! Our free tier and low usage tiers are as fast, as available, and as secure as our at scale service. You get all our availability features like multi-AZ replication, hot key protection, and automatic scaling to handle bursts. You even get all our security features for free (end-to-end encryption, per request authentication, TLS).
+<summary>Momento を無料で本番環境のアプリケーションに使うことはできますか？</summary>
+もちろんです！無料枠や低い利用率でも、大規模利用のものと同様に、高速で可用性がありセキュアです。複数AZ レプリケーション、ホットキー保護、そして突発的なリクエストへの自動スケーリングといった高可用性のための機能が全て利用可能です。セキュリティのための全ての機能(エンドツーエンド暗号化、リクエスト毎の認証、TLS)も無料で使えます。
 
-We frequently find customers provisioning an entire cluster to handle low-RPS workflows. If you want them to be HA, you need multiple nodes. If you believe in CICD, you likely have a similarly sized cluster in staging—and maybe even in dev. That adds up! Turn off those machines—and save the environment. Go Momento!
+お客さんの中でよく、低いRPS のワークフローのためにフル装備のクラスターをプロビジョンしているケースを見かけます。HA (高可用性)が欲しい場合、複数ノードが必要です。CICD を利用したければ、同程度のサイズのクラスターを、ステージングや開発環境にさえ設置したくなるでしょう。全てコストとして積み上がります！こうしたマシンは停止してしまって、コストを削減しましょう。Momento で行きましょう！
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/pricing.md]: お客様

File: i18n/ja/docusaurus-plugin-content-docs/current/pricing.md [hunk 4]
```diff
@@ -2,76 +2,70 @@
...
+はい、もちろんです！Momento は将来も使い続けられるキャッシュとして最高の選択肢です。Momento を追加するのは *ものすごく* 素早くできて、そのあとは1RPS だろうが100万RPS だろうが、あとのことは全てバックグラウンドに行われるので皆さんが考える必要がありません。使った分だけ料金を払うだけでよいので、スケール可能なキャッシュを高額な料金を支払うことなく得られます。
 </details>
   
-<details>
-  <summary>Will it blend?</summary>
-We do not recommend putting Momento in a blender as it may void the warranty of your blender, but Momento is robust with enterprise grade security and availability. Momento does blend seamlessly with your current cloud setup, whether you're on AWS, GCP, Azure, or multi-cloud!
-</details>
-
-[Give Momento a Try!](getting-started.md)
+[Momento をはじめてみましょう！](getting-started.md)
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/pricing.md]: Not sure why but the link is not displayed properly...

image

Comment (riywo) [reply i18n/ja/docusaurus-plugin-content-docs/current/pricing.md]: Sorry, this was caused by unnecessary whitespaces on the previous line. Fixed by the new commit.

**Comment by riywo** [comment]
@poppoerika Addressed your comments. Please review it again!

**Review by poppoerika — APPROVED** [review]
🥳

**Pull request merged**